src/file-utils.js

```diff
@@ -1,60 +1,69 @@
-import {exec} from "child_process";
```

Comment:
If you've changed this, just to be able to patch this in tests, you don't need to...

Raw:

```js
import {exec} from "child_process";

exec();
```

babel output:

```js
var _child_process = require("child_process");

(0, _child_process.exec)();
```

The same applies after refactoring to sinon (for `sandbox.stub(childProcess, "exec")`) where it would still work.